domenetjenester/fpsak/src/main/java/no/nav/foreldrepenger/fpsak/dto/behandling/familiehendelse/FamiliehendelseDto.java

```diff
@@ -0,0 +1,41 @@
+package no.nav.foreldrepenger.fpsak.dto.behandling.familiehendelse;
+
+import java.time.LocalDate;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME)
+@JsonSubTypes({
+        @JsonSubTypes.Type(value = AvklartDataFodselDto.class),
```

Comment:
bør også sette eksplisitt name på JsonSubTypes.Type (mindre sårbar for refaktorering enn default classenavn)